_data/faq.yml

```
@@ -115,10 +115,127 @@ it:
+      of art. 69 regarding the choice of the license?
+    answer: |
+      The "Guidelines on the acquisition and reuse of software for the public
+      administrations", adopted by AgID in implementation of Articles 68 and 69
+      of the CAD, foresee that the choice of the license by a PA is
+      made by choosing a license among those certified by the Open Source
+      Initiatives. For more information about licensing see <a
+      href="https://docs.italia.it/italia/developers-italia/gl-acquisition-and-reuse-software-for-pa-docs/en/stabile/software-reuse/open-licences-and-selecting-a-licence.html#choosing-a-licence">this
+      resource</a>.
+  item6:
+    question: Is it possible in the process of Reuse of a solution
+      apply, by virtue of the principle of compatibility of licenses, to the work
+      derived a different license from the original work?
+    answer: |
+      The "Guidelines on the acquisition and reuse of software for the public
+      administrations", adopted by AgID in implementation of Articles 68 and 69
+      CAD, define, in paragraph 3.5.3, the decision tree for the
+      choice of an open license; in particular, if the release of the software
+      refers to a modification of existing open source software (so
+      taken for reuse by other administrations or owned by third parties)
+      the administration must use the same license with which the software has
+      been originally distributed, in order to facilitate its maximum
+      interoperability and reuse with other users of the same software.
+  item7:
+    Question: To which authorities can be reported the
+      non-application by a PA of articles 68 and 69 of CAD?
+    answer: |
+      According to art. 17, paragraph 1-quater is established by the AgID the
+      Office of the Digital Ombudsman who has the task of collecting all reports
+      of supposed violations of the Code of the Digital Administration, or any
+      other rule on the subject of digitization and innovation, to guarantee
+      the digital rights of the citizens and businesses.
+      Anyone can report supposed violations of CAD provisions,
+      among which those ex art. 68 and 69, accessing the appropriate <a
+      href="https://difensorecivicodigitale.gov.it/index.php?a=add">area
+      dedicated</a> on AgID website.
+  item8:
+    question: What fines could be imposed at a PA that acquires solutions,
+      without making a technical and economic comparison?
+    answer: |
+      The CAD does not provide direct fines; however, if a PA disregards the
+      provisions concerning the acquisition of software may arise management
+      and disciplinary responsibility profiles without prejudice to any
+      liability for damages treasury.
+  item9:
+    question: What fines could be imposed to a PA if it not proceed to the release
+    in open source of the software developed or commissioned by it?
```

```suggestion
      in open source of the software developed or commissioned by it?
```
I guess there's an issue with spacing here